inline_data.h

```diff
@@ -0,0 +1,230 @@
+const char str_printf[] = "[%d]\n";
```

Comment:
A comment for what this file represents, and how all these numbers were obtained will be *really* nice.